TLC-88: Put Family Name first in all queues

ouiliam opened the pull request:
modified:   configuration/globalproperties/gp.xml

Commit:
TLC-88: Put Family Name first in all queues
	modified:   configuration/globalproperties/gp.xml

rbuisson commented:
@ouiliam I am surprised of the format you used in the file. Does the import correctly work as it is?

ouiliam commented:
@rbuisson yes, the current contents of default sqlsearches are formatted this way on prod too, much more readable

Commit:
TLC-88: Correct and Complete Lab Orders

	modified:   configuration/concepts/laborders.csv
	modified:   configuration/globalproperties/gp.xml
	modified:   configuration/scripts/post_run/configureOrderTypes.sql

ouiliam committed:
Update configureOrderTypes.sql

ouiliam committed:
Update configureOrderTypes.sql

rbuisson commented:
Merged
